dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -221,14 +219,7 @@ private String readFileFromRepo(String fileName, String reference, GHRepository
             // * All repositories from organizations I belong to
 
             final int pageSize = 30;
-            Map<String, GHRepository> allMyRepos = new TreeMap<>();
-            github.getMyself().listRepositories(pageSize, GHMyself.RepositoryListFilter.ALL).forEach((GHRepository r) -> allMyRepos.put(r.getFullName(), r));
-
-            Map<String, GHRepository> allRepositories = Collections.unmodifiableMap(allMyRepos);
-
-            for (Map.Entry<String, GHRepository> innerEntry : allRepositories.entrySet()) {
-                reposByGitURl.put(innerEntry.getValue().getSshUrl(), innerEntry.getValue().getFullName());
-            }
+            github.getMyself().listRepositories(pageSize, GHMyself.RepositoryListFilter.ALL).forEach((GHRepository r) -> reposByGitURl.put(r.getSshUrl(), r.getFullName()));
```

Comment:
The more purist functional way would be to return a map directly instead of pushing into a variable that exists outside of the function.

Something like the first example from [here](https://www.mkyong.com/java8/java-8-convert-list-to-map/).

Just FYI; I'm still learning functional myself, and am not purist about it. :)